nexus/db-queries/src/db/datastore/snapshot.rs

```diff
@@ -93,6 +93,11 @@ impl DataStore {
                     // If this function is passed a snapshot with an ID that
                     // does not match, but a project and name that does, return
                     // ObjectAlreadyExists here.
+                    //
+                    // NOTE: an earlier version of this comment suggested using
+                    // filter_target for time_deleted, which does not work with
+                    // CockroachDB:
+                    // https://github.com/oxidecomputer/omicron/issues/5047
```

Comment:
is this really necessary? i feel like this information is kind of captured in the Git history already...

Reply:
removed! (Honestly I was trying to wrap my head around this)